src/bp-members/bp-members-functions.php

```diff
@@ -5346,25 +5346,38 @@ function bb_user_presence_html( $user_id, $expiry = true ) {
  *
  * @since BuddyBoss 2.3.1
  *
```

Comment:
```suggestion
 *@since BuddyBoss [BBVERSION] The `$force` parameter was added.
```

Now onwards we should start adding multiple since tags if we make any changes to existing functions.